spec/features/visiting_articles_index_spec.rb

```diff
@@ -1,10 +1,11 @@
-#commenting out until pge is complete
+require 'rails_helper'
 
-# require 'rails_helper'
+feature "visiting the article index page" do
+  scenario "The user can visit the page that lists all the articles" do
+    visit "/articles"
+    expect(page).to have_content "All Articles"
```

Comment:
This only tests that a header is displayed, so the test name may be a bit misleading.